usr/lib/tik/modules/pre/10-welcome

```diff
@@ -1,5 +1,39 @@
 # SPDX-License-Identifier: MIT
 # SPDX-FileCopyrightText: Copyright 2024 SUSE LLC
-# SPDX-FileCopyrightText: Copyright 2024 Richard Brown
+# SPDX-FileCopyrightText: Copyright 2024 Raymond Yip
+
+proceedInstall() {
+	d --info --ok-label="Install Now" --no-wrap --width=300 --height=300 --icon=distributor-logo-Aeon-symbolic --title="" --text="<big>Welcome to ${TIK_OS_NAME}</big>\n\nPlease press <b>Install Now</b> to continue"
+}
+
+warningMSG() {
+	d --warning --ok-label="OK" --no-wrap --width=300 --height=300 --icon=distributor-logo-Aeon-symbolic --title="" --text="Please connect the AC power adapter to your machine to continue" 
```

Comment:
```suggestion
	d --warning --no-wrap --title="AC Power Required" --text="Please connect the AC power adapter to your machine to continue" 
```
Fixed dimensions aren't needed, a title makes it a lot clearer why the message is happening, the ok-label is OK by default, and it's better to use the default warning icon